### ISSUE TITLE
Python model to javascript model always return same prediction

### ISSUE THREAD

**Issue by danielreyes9756** [tensorflow/tensorflow#48710]
Hi and thanks in advance,

I create my own model in python and I get a 93% of accuracy,
then I convert my model to a js model using the command "tensorflowjs_converter --input_format keras customModel_gs.h5 model/", 

I add the model file to my angular project and then I try this:
![imagen](https://user-images.githubusercontent.com/43205522/115792840-e2dc3f80-a3c2-11eb-80a7-0a3d9f0e7861.png)

The model charge I can do a summary all is correct except the predition, doesn't care the img a upload, always return the same number 2.

My input_shape is a null,128,128,1 (grayscale), I have 4 classes
![imagen](https://user-images.githubusercontent.com/43205522/115793218-a78e4080-a3c3-11eb-9df2-20f3388335dc.png)
![imagen](https://user-images.githubusercontent.com/43205522/115793289-c391e200-a3c3-11eb-9b31-19ec655ea870.png)

**Comment by UsharaniPagadala** [comment]
@danielreyes9756 

Could you please fill the issue template

In order to expedite the trouble-shooting process, could you please provide the following details

OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
TensorFlow installed from (source or binary):
TensorFlow version:
Python version:
Installed using virtualenv? pip? conda?:
Bazel version (if compiling from source):
GCC/Compiler version (if compiling from source):
CUDA/cuDNN version:
GPU model and memory
and the exact sequence of commands / steps that you executed before running into the problem. Thanks!

**Comment by danielreyes9756** [comment]
Yeah of course thanks,
OS Windows 10
issue happens on PC device, im working on angular 11
model crearted on google colaboratory so:
TensorFlow default version and intallation of google colaboratory 
Same with Python 3.7.10 default version I didn't install it
Bazel version (I don't know what is Bazel)
Not need CUDA
GPU model and memory, I workerd with the colab gpu acelerator but my own GPU is a GTX 1650 4GB GDDR6

In colab the only command I use where 
model.save("route/(customModel_gs.h5")
!pip install tensorflowjs
!tensorflowjs_converter --input_format keras customModel_gs.h5 model/",

**Comment by Saduf2019** [comment]
@danielreyes9756 
As this is tfjs issue, could you please open this in the tfjs repo and move this to closed status.
